packages/loader/container-loader/src/index.ts

```diff
@@ -10,3 +10,5 @@ export * from "./loader";
 export * from "./networkUtils";
 export * from "./utils";
 export * from "./retriableDocumentStorageService";
+export * from "./containerContext";
+export * from "./connectionStateHandler";
```

Comment:
does this really need to be exported? is it ever made public?

Reply:
if only ConnectionState needs to be public maybe but that somewhere else. its own file would even be fine

---
In reply to: [595614005](https://github.com/microsoft/FluidFramework/pull/5542#discussion_r595614005) [](ancestors = 595614005)

Reply:
best not to export things publicly that shouldn't be used publiclly

---
In reply to: [595614572](https://github.com/microsoft/FluidFramework/pull/5542#discussion_r595614572) [](ancestors = 595614572,595614005)

Reply:
Removed.